src/docs/markdown/service.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Linux Service
+---
+
+Linux Service
```

Comment:
Make a new h1, "System service", and then make this a h2, and all the h2's into h3's, etc.